crates/bevy_graph/src/error.rs

```diff
@@ -0,0 +1,12 @@
+use crate::graphs::keys::{EdgeIdx, NodeIdx};
+
+#[derive(Debug)]
```

Comment:
```suggestion
#[derive(Debug, Error)]
```

Use the `thiserror` crate.